rubicon/objc/api.py

```diff
@@ -649,6 +656,13 @@ def __new__(cls, object_ptr, _name=None, _bases=None, _ns=None):
 
         return self
 
+    def __del__(self):
+        # Autorelease all objects which we own, except for autorelease pools.
+        # We use autorelease instead of release here to prevent instances without a Python
+        # reference but with an Obj-C reference from being released too early.
+        if self._needs_release and self.objc_class.name != 'NSAutoreleasePool':
+            self.autorelease()
```

Comment:
It might be a good idea to use `send_message` for this call instead of the normal high-level syntax, to avoid the overhead of Rubicon looking up the method first, and also saving an unnecessary wrapping of the returned object (which is just `self`). With any luck it could also change something about the segfault.